### DISCUSSION TITLE
Add multiple examples for 1 response

### DISCUSSION THREAD

**Discussion by arthurvanl** [scalar/scalar#3968] [Ideas]
on status 200 I have multiple schemas depending on query parameters or different values of properties (e.g. request body). Thus giving me the problem of not correctly showing what the response is of 200

**Comment by hanspagel** [comment]
Did you try oneOf? Or can you share your OpenAPI document, or the relevant part with us?

**Reply by arthurvanl** [reply]
Yes I tried:

![image](https://github.com/user-attachments/assets/ee87e4c0-bfea-48c4-b568-5a5e711c7208)

![image](https://github.com/user-attachments/assets/6be135d8-deb2-4804-83b0-8c8b7e8c56f2)
as you can see they are both there, but I would like to be able to switch from examples.
